common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -4040,8 +4042,8 @@ object CelebornConf extends Logging {
       .createWithDefault(false)
 
   val CLIENT_PUSH_SORT_SMALL_PUSH_TOLERATE_FACTOR: ConfigEntry[Double] =
-    buildConf("celeborn.client.spark.push.sort.smallPushTolerateFactor")
-      .withAlternative("celeborn.push.sortMemory.adaptiveThreshold")
+    buildConf("celeborn.client.spark.push.sort.memory.smallPushTolerateFactor")
+      .withAlternative("celeborn.push.sortMemory.smallPushTolerateFactor")
```

Comment:
the alternative conf key is unnecessary.

Reply:
updated